.gitmodules

```diff
@@ -43,3 +40,12 @@
 	url = git@github.com:NVlabs/cub.git
 [submodule "Submod"]
 	branch = rljacob/e3smconfig/elm-name
+[submodule "components/mpas-ocean/src/cvmix"]
+	path = components/mpas-ocean/src/cvmix
+	url = https://github.com/CVMix/CVMix-src.git
```

Comment:
@mark-petersen , the three new ocean code submodules have been added using the https protocol, but all other E3SM submodules use the ssh protocol.  Is that a mistake?

Reply:
OK, I changed it to git@github now and it all works. Thanks.